mailer.js

```diff
@@ -250,25 +278,30 @@ module.exports = function (log) {
         'X-Verify-Code': message.code
       },
       subject: gettext('Confirm new sign-in to Firefox'),
-      template: 'verifyLoginEmail',
+      template: templateName,
       templateValues: {
+        alternativeLink: alternativeLink,
         device: this._formatUserAgentInfo(message),
         email: message.email,
         ip: message.ip,
         link: link,
         location: this._constructLocationString(message),
         oneClickLink: oneClickLink,
-        passwordChangeLink: this.createPasswordChangeLink(message.email),
-        passwordChangeLinkAttributes: this._passwordChangeLinkAttributes(message.email),
-        supportLinkAttributes: linkAttributes(this.supportUrl),
-        supportUrl: this.supportUrl,
+        passwordChangeLink: this.createPasswordChangeLink(message.email, templateName),
+        passwordChangeLinkAttributes: this._passwordChangeLinkAttributes(message.email, templateName),
+        privacyUrl: this.createPrivacyLink(templateName),
+        supportLinkAttributes: this._supportLinkAttributes(templateName),
+        supportUrl: this.createSupportLink(templateName),
         timestamp: this._constructLocalTimeString(message.timeZone, message.acceptLanguage)
       },
       uid: message.uid
     })
   }
 
   Mailer.prototype.recoveryEmail = function (message) {
+    // TODO: There seems to be some discrepancy in email template names, some places
```

Comment:
Ahha, that's the answer to [this question above](https://github.com/mozilla/fxa-auth-mailer/pull/190/files/8ec8d82a8b88556c4379fa038e8c06db65a4a953#r73328105). Time to unify nomenclature?